app/shop.py

```diff
@@ -0,0 +1,12 @@
+import dataclasses
+from typing import Any
+
+
+@dataclasses.dataclass
+class Shop:
+    name: str
+    location: tuple
+    products: dict
+
+    def prod_price(self) -> Any:
+        return self.products.values()
```

Comment:
Remove this function and just use:
shop.products.values()
No need to create unneeded function